src/map_command_handler.hpp

```diff
@@ -201,7 +201,7 @@ class map_command_handler
 			std::string string_user = get_cmd();
 			int distance = 0;
 			// Minimum length of the two compared strings.
-			int len_min = 0;
+			std::size_t len_min = 0;
```

Comment:
`distance` and `len_min` are never accessed outside the `if` block below, so they can just be declared as `const auto` at the point that they're first assigned a meaningful value.

Reply:
Updated.